finesse/hardware/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""This module contains code for interfacing with different hardware devices."""
+from .dummy_stepper_motor import DummyStepperMotor
+
+# TODO: Replace with a real stepper motor device
+stepper = DummyStepperMotor(3600)
```

Comment:
I don't think we want to create a global stepper here. We need to think about this, but considering how many devices we have, I think we should consider to setup a sort of `DeviceManager` living at the same level as the MainWindow in charge of starting them, dealing with their Errors, etc. For now, it is ok, but something to bear in mind.

Reply:
Yep, I agree. Once the rest of the code takes shape it'll become a bit more obvious how to approach this.